napalm_junos/junos.py

```diff
@@ -999,23 +999,23 @@ def _bgp_iter_core(neighbor_data, instance=None):
                 neighbor_details.update(neighbor_rib_details)
                 bgp_neighbors[instance_name][remote_as].append(neighbor_details)
 
-        old_junos = napalm_base.helpers.convert(
-            int, self.device.facts.get('version', '0.0').split('.')[0], 0) < 15
+        # old_junos = napalm_base.helpers.convert(
+        #     int, self.device.facts.get('version', '0.0').split('.')[0], 0) < 15
         bgp_neighbors_table = junos_views.junos_bgp_neighbors_table(self.device)
 
-        if old_junos:
-            instances = junos_views.junos_route_instance_table(self.device)
-            for instance, instance_data in instances.get().items():
-                if instance.startswith('__'):
-                    # junos internal instances
-                    continue
-                neighbor_data = bgp_neighbors_table.get(instance=instance,
-                                                        neighbor_address=neighbor_address).items()
-                _bgp_iter_core(neighbor_data, instance=instance)
-        else:
-            bgp_neighbors_table = junos_views.junos_bgp_neighbors_table(self.device)
-            neighbor_data = bgp_neighbors_table.get(neighbor_address=neighbor_address).items()
-            _bgp_iter_core(neighbor_data)
+        # if old_junos:
+        instances = junos_views.junos_route_instance_table(self.device)
+        for instance, instance_data in instances.get().items():
+            if instance.startswith('__'):
+                # junos internal instances
+                continue
+            neighbor_data = bgp_neighbors_table.get(instance=instance,
+                                                    neighbor_address=neighbor_address).items()
+            _bgp_iter_core(neighbor_data, instance=instance)
+        # else:
+        #     bgp_neighbors_table = junos_views.junos_bgp_neighbors_table(self.device)
```

Comment:
I thought about leaving them for later versions when hopefully the `peer_fwd_rti` key will be available on all Junos distributions.
But we can remove, no problem.